recipes/zbarlight/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "zbarlight" %}
+{% set version = "2.0" %}
+{% set sha256 = "7fce62b0240a1770ea4174eb4593a06351f391a028cce0d80fa7b9c74c7c6607" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - zbar
+    - pillow
+  run:
+    - python
+    - pip
+    - setuptools
```

Comment:
You'll need setuptools only if this package uses pkg_resources, can you confirm that?

Reply:
it seems to, yes.